kong/db/schema/entities/routes.lua

```diff
@@ -37,20 +37,20 @@ if kong and kong.configuration and kong.configuration.router_flavor == "atc" the
       { response_buffering  = { type = "boolean", required = true, default = true }, },
       { tags             = typedefs.tags },
       { service = { type = "foreign", reference = "services" }, },
-      { atc = { type = "string", required = true }, },
+      { expressions = { type = "string", required = true }, },
```

Comment:
`expressions` or `expression`? since it is a `type=string` field.

Reply:
rename to `expression`